QualtricsIntegration/README.md

```diff
@@ -0,0 +1,52 @@
+# Qualtrics Survey Operators #
+The custom operators provided in this sample are designed to extract responses from **Qualtrics**.
+
+Coding was done on Python 3.6.x and was tested with Data Hub version 2.7.
+
+## Requirements
+
+Before you start using the example, please make sure that:
+
+- You are familiar with the basic concepts of SAP Data Hub Modeling such Pipelines (Graphs), Operators and Dockerfiles.  For more information, you may refer to the Modeling Guide for SAP Data Hub that is available on the SAP Help Portal (https://help.sap.com/viewer/p/SAP_DATA_HUB).
+- You are familiar with the basic concepts of Docker (https://docs.docker.com/get-started/) and Kubernetes (https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/).
+
+## Content
+**1. Dockerfile 'qualtrics' ([Source code](src/qualtrics-1.0.0/content/files/vflow/dockerfiles/com/poc/qualtrics/qualtrics/Dockerfile))**
+  - Extends the SAP-delivered Dockerfile com.sap.python27
```

Comment:
It uses opensuse-leap:15.0-sap24, not the python27 image

Reply:
corrected

Reply:
looks good